src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/TranslatedDocumentUploadedClaimantNotificationHandler.java

```diff
@@ -61,18 +68,34 @@ CLAIM_LEGAL_ORG_NAME_SPEC, getApplicantLegalOrganizationName(caseData)
 
     private CallbackResponse notifyClaimant(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
-        if (caseData.isLipvLipOneVOne() && featureToggleService.isLipVLipEnabled()) {
-            return AboutToStartOrSubmitCallbackResponse.builder().build();
+        String email = addEmail(caseData);
+        if (Objects.nonNull(email)) {
+            notificationService.sendMail(
+                email,
+                addTemplate(caseData),
+                addProperties(caseData),
+                String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference())
+            );
         }
-        notificationService.sendMail(
-            caseData.getApplicantSolicitor1UserDetails().getEmail(),
-            notificationsProperties.getNotifyClaimantTranslatedDocumentUploaded(),
-            addProperties(caseData),
-            String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference())
-        );
         return AboutToStartOrSubmitCallbackResponse.builder().build();
     }
 
+    private String addTemplate(CaseData caseData) {
+        if (caseData.isLipvLipOneVOne() && featureToggleService.isLipVLipEnabled()) {
+            if (caseData.isBilingual()) {
+                return notificationsProperties.getNotifyClaimantLiPTranslatedDocumentUploadedWhenClaimIssuedInBilingual();
+            }
+            return notificationsProperties.getNotifyClaimantLiPTranslatedDocumentUploadedWhenClaimIssuedInEnglish();
+        }
+        return notificationsProperties.getNotifyClaimantTranslatedDocumentUploaded();
+    }
+
+    private String addEmail(CaseData caseData) {
```

Comment:
can you please rename this method. This is not adding email anywhere.

Reply:
done